docs/pages/repo/docs/core-concepts/monorepos/ignoring.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Ignoring tasks
+description: Never do the same work twice.
+---
+
+# Ignoring Tasks
+
+[Build caching](/repo/docs/core-concepts/caching) can dramatically speed up your build tasks - but you can do even better by using `npx turbo-ignore`. If your workspace is unaffected by your code changes, you can completely ignore executing a task altogether.
+
+## Using `turbo-ignore`
+
+To skip unaffected work with `npx turbo-ignore`, all you'll need is to make sure your git history is available.
+
+By default, `turbo-ignore` will run the build task in your current working directory. Under the hood, `turbo-ignore` compares the output of `--dry=json` from your previous commit to your current commit.
```

Comment:
This depends - when deploying on Vercel, it won't compare against your previous commit by default, it will compare against the previously deployed SHA.